src/chat.rs

```diff
@@ -2585,7 +2600,7 @@ pub async fn set_chat_name(
                 .await
                 .is_ok()
             {
-                if chat.is_promoted() {
+                if chat.is_promoted() && !chat.is_mailing_list() {
```

Comment:
changing the name locally but not sync it would raise ux problems, see recent discussions at https://github.com/deltachat/deltachat-android/pull/1772#pullrequestreview-573124262 . i would move this check up as `ensure!(!chat.is_mailing_list(), "Mailing lists can't be changed");`

(same a few lines below in `set_chat_profile_image()`)

Reply:
As written [above](https://github.com/deltachat/deltachat-core-rust/pull/1964#discussion_r562843998) I think we should think again about not letting the user change the name as sometimes it's very unfitting and they may also change the name of 1:1 chats without expecting that this change is sent out

Reply:
i am not against thinking that over, however, maybe not do that in scope if this pr - as this would involve multi-device-sync and ux questions.

Reply:
Well we don't sync changed contact names between devices, either

Reply:
I also think that allowing to change mailing list group name is fine as long as no device message is shown it is clear no message went out

Reply:
kk :)